binding/web/src/rhino.ts

```diff
@@ -305,76 +268,47 @@ export class Rhino {
    * environment with overlapping speech (e.g. people talking in the background).
    * @param options.processErrorCallback User-defined callback invoked if any error happens
    * while processing the audio stream. Its only input argument is the error message.
-   * @param options.customWritePath Custom path to save the model in storage.
-   * Set to a different name to use multiple models across `rhino` instances.
-   * @param options.forceWrite Flag to overwrite the model in storage even if it exists.
-   * @param options.version Rhino model version. Set to a higher number to update the model file.
    *
    * @returns An instance of the Rhino engine.
    */
-  public static async fromPublicDirectory(
+  public static async create(
     accessKey: string,
     context: RhinoContext,
     inferenceCallback: InferenceCallback,
-    publicPath: string,
+    model: RhinoModel,
     options: RhinoOptions = {}
   ): Promise<Rhino> {
-    const {
-      customWritePath = 'rhino_model',
-      forceWrite = false,
-      version = 1,
-      ...rest
-    } = options;
-    await fromPublicDirectory(customWritePath, publicPath, forceWrite, version);
-    const contextPath = await contextProcess(context);
-    return this.create(
+    if (
+      context.customWritePath == null ||
+      context.customWritePath == undefined
+    ) {
+      context.customWritePath = 'rhino_context';
+    }
+    const contextPath = await loadModel(context);
+    const { sensitivity = 0.5 } = context;
+
+    if (model.customWritePath == null || model.customWritePath == undefined) {
+      model.customWritePath = 'rhino_model';
+    }
+    const modelPath = await loadModel(model);
```

Comment:
we modify the original object this way, how about:
```suggestion
  const customWritePath = (model.customWritePath) ? model.customWritePath : 'rhino_model';
  const modelPath = await loadModel({ ...model, customWritePath});
```